src/www/pages/styles/maps/index.njk

```diff
@@ -0,0 +1,176 @@
+{% extends "www/_layouts/layout-left-pane.njk" %}
+{% set pageTitle = "Maps" %}
+{% set section = "Styles" %}
+
+{# Imports #}
+{%- from "wmnds/components/link/link/_link.njk" import wmndsLink -%}
+
+{% block content %}
+{# ESRI ArcGIS #}
+<h2>ESRI ArcGIS</h2>
+
+{# About #}
+<h3>About</h3>
+<p>
+  When it comes to mapping, West Midlands Network uses
+  {{
+  wmndsLink({
+    link: "https://developers.arcgis.com/labs/what-is-arcgis/",
+    linkTitle: "ESRI ArcGIS products",
+    linkTarget: "_blank",
+    text: "ESRI ArcGIS products"
+    })
+  }}
+  to create, manage and share data on maps. Therefore, when developing maps use
+  {{
+    wmndsLink({
+      link: "https://developers.arcgis.com/documentation/core-concepts/apis-sdks-apps/#choosing-an-arcgis-api-or-sdk",
+      linkTitle: "ArcGIS APIs and SDKs",
+      linkTarget: "_blank",
+      text: "ArcGIS APIs and SDKs"
+    })
+  }}.
+</p>
+
+{# Basemap #}
+<h3>Basemap</h3>
+  <a href="//disruptions.wmnetwork.co.uk/" class="wmnds-col-1" target="_blank" rel="noopener noreferrer">
+    <img class="wmnds-col-1 wmnds-m-b-md" src="$*cdn/img/component-images/maps/base-map.png" alt="A screenshot showing an example of a map">
```

Comment:
This image it's not loading in the preview page, is that okay? Or does source has any typo? (not sure how cdn works here)

Reply:
@catiarodriguescosta This should be resolved in my last commit.